cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -152,6 +152,10 @@ public BuildCommand(Path projectPath, PrintStream outStream, PrintStream errStre
     @CommandLine.Option(names = "--taint-check", description = "perform taint flow analysis")
     private Boolean taintCheck;
 
+    @CommandLine.Option(names = "--include-all", hidden = true,
```

Comment:
Can we make this useful for general users ? without just having a hidden option

Reply:
For that, we need to able to get the information in the test report as well. That would need to come up with a design for the report.